app/controllers/application_controller.rb

```diff
@@ -120,6 +122,10 @@ def authenticate_user!
     end
   end
 
+  def set_default_view_variables
+    @adv_search = Query::Proposal::FieldedSearch.new({})
```

Comment:
do we want to make search available on *every* page? 

Right now, for example, search is not available on Proposals#show but it is available on Proposals#show with this PR. Not bad to have it there, just not sure if that was intended. 

Placement on proposals#show does not make it super obvious that the search is for any proposal rather than within the current proposal:

![screen shot 2016-02-01 at 10 55 12 am](https://cloud.githubusercontent.com/assets/601515/12727494/2f1efc88-c8d2-11e5-96bf-4c38b791a875.png)